packages/shared-core/src/filters.ts

```diff
@@ -111,7 +117,8 @@ const removeKeyNumbering = (key: string) => {
 }
 
 type Filter = {
-  operator: keyof Query
+  operator: keyof Omit<Query, "onEmptyFilter">
+  onEmptyFilter: EmptyFilterOption
```

Comment:
This should be optional - it'll only exist on at most one filter.

Reply:
Made optional